keywording/text.xml

```diff
@@ -434,6 +434,36 @@ keyword, so packages are not to be marked stable for one of these architectures.
 </body>
 </subsection>
 
+<subsection>
+<title>Keeping track of pending stabilizations</title>
+<body>
+
+<p>
+Maintainers need some method or system to organize and start pending
+stabilizations once they become due.
+</p>
+
+<p>
+There are several options available:
+</p>
+
+<ul>
+  <li>
+    Use <c>imlate</c> from app-portage/gentoolkit
```

Comment:
Maybe a comment saying that imlate appears unmaintained and doesn't understand projects?

Reply:
Oh, it does work, it just needs explicit email via -M.